website/README.md

```diff
@@ -98,31 +101,25 @@ Stop the postgresql service using
     pg_ctl -D /usr/local/var/postgres stop  # if not installed using homebrew
     
 ### Linux
-**_NOTE_**:
 
-Currently the environment does not run well on Windows Bash / WSL ( Windows Subsystem for Linux ).
```

Comment:
@FaithDaka , I am abit anxious about these deletions from the README file.
Could you add some PR comments perhaps? Just to guide the reviewers? This will also remove a need for a meeting for you to explain the changes.

Thanks!

Reply:
Adding a separate instruction line for Linux users